Hogajama/hogajama-ai/src/main/java/com/gepardec/hogarama/ai/Function.java

```diff
@@ -0,0 +1,31 @@
+package com.gepardec.hogarama.ai;
+
+public class Function {
+    private String name;
+    private String description;
+    private Object parameters;
```

Comment:
Ist das für uns nicht eine `Map<String, String>`?
Eher doch nicht. In der OpenAI Doku:
```
public Object getParameters()

Get the parameters property: The parameters the functions accepts, described as a JSON Schema object.
```

Reply:
Generell nein, weil jedes Value-Objekt kann wieder ein Map oder eine Zahl oder was auch immer sein.